float64 Position1
float64 Position2
float64 Position3
float64 Position4

float64 Velocity1
float64 Velocity2
float64 Velocity3
float64 Velocity4

float64 Torque1
float64 Torque2
float64 Torque3
float64 Torque4
